core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -455,6 +459,21 @@ public ServerBuilder localPort(int port, Iterable<SessionProtocol> protocols) {
         return this;
     }
 
+    /**
+     * Adds the specified {@link AppInfo} that represents information about a deployed application via
+     * {@link ManagementService}.
+     * <pre>{@code
+     * ServerBuilder sb = Server.builder();
+     * sb.setAppInfo(new AppInfo("1.0.0", "name", "description"));
+     * }
+     * </pre>
+     */
+    public ServerBuilder setAppInfo(AppInfo appInfo) {
```

Comment:
Should we also expose `AppInfo` to Armeria's documentation service?
`Application information` table could be added if `AppInfo` exists.
https://github.com/line/armeria/blob/e1d137738a8e4f3f7a7943e69f3f314ab772e338/docs-client/src/containers/HomePage/index.tsx#L95-L99